Makefile.toml

```diff
@@ -7,22 +7,23 @@ VERBOSE = ""
 TLS_PROVIDER = "rustls"
 TARGET = ""
 WORKSPACE = "${CARGO_MAKE_WORKING_DIRECTORY}"
+MSRV = ""
 
 [env.release]
 BUILD_FLAVOR = "--release"
-RUSTFLAGS="-C link-arg=-fuse-ld=lld -C codegen-units=1 -C panic=abort -C lto=fat -C opt-level=3"
```

Comment:
Maybe in the future we can experiment with using the Mold linker if we really care about linking performance

Reply:
I've been using it for a while, makes a big difference